DNN Platform/Library/Services/Personalization/Personalization.cs

```diff
@@ -2,22 +2,27 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
 namespace DotNetNuke.Services.Personalization
-{
+{
+    using System;
     using System.Web;
 
     using DotNetNuke.Common.Utilities;
     using DotNetNuke.Entities.Portals;
     using DotNetNuke.Entities.Users;
     using DotNetNuke.Security;
+    using static DotNetNuke.Entities.Portals.PortalSettings;

```

Comment:
Doesn't look like this is in use
```suggestion
    using static DotNetNuke.Entities.Portals.PortalSettings;
```

Reply:
Yes it is used for `PortalSettings.Current.DefaultControlPanelMode`